Sources/MenuViewController.swift

```diff
@@ -18,12 +18,45 @@
 
 import UIKit
 
-open class MenuViewController: UIViewController {
+@objc public protocol MenuViewController
+{
+    // UIViewController
+    var transitioningDelegate: UIViewControllerTransitioningDelegate? { get set }
+    var view: UIView! { get }
+    
+    // MenuViewController
+    weak var menuContainerViewController: MenuContainerViewController? { get set }
+    var navigationMenuTransitionDelegate: MenuTransitioningDelegate? { get set }
+    @objc func handleTap(recognizer: UIGestureRecognizer)
+}
 
-    public weak var menuContainerViewController: MenuContainerViewController?
-    var navigationMenuTransitionDelegate: MenuTransitioningDelegate?
+private var key_menuContainerViewController = "menuContainerViewController"
```

Comment:
@truemetal Currently, we try to avoid using Obj-C Runtime and keep our library on the pure swift.